data/autoyast_sle15/create_hdd/create_hdd_maintenance.xml.ep

```diff
@@ -210,6 +210,15 @@
 cd /etc/zypp/repos.d
 sed -i 's/enabled=1/enabled=0/' $(ls|grep -i nvidia)
 zypper lr
+
+# Regarding rpm lock see Q9@ https://documentation.suse.com/de-de/sles/15-SP1/html/SLES-all/app-ay-faq.html
+# quote : "during the post-script phase, YaST still has an exclusive lock on the RPM database."
+# the package cannot be installed the autoyast way, because of the changing package name (eg libyui-rest-api15)
+# only zypper allows to install "by capability".
+mv /var/run/zypp.pid /var/run/zypp.sav
+zypper -n in libyui-rest-api
```

Comment:
we will need a a couple of conditions with an 'or' in the middle with ERB to only run it for SLE-15-SP3 and SLE-15-SP4 (to be soon settled in maintenance). There is not libyui-rest-api for older products.
We are assuming that we will reuse the same profile even for SLE-15-SP2, that is why i wanted to schedule later (not sure how you took it so soon, maybe my bad :) but thanks for taking a look so soon anyway. Could you please add them, otherwise, we just need to mention this in the follow-up tickets for creating AutoYaST images in older products.

Reply:
Condition added.

Reply:
we can do regex! that is even better! :)